src/commands/sourcemaps/inject.rs

```diff
@@ -56,24 +56,15 @@ fn fixup_files(paths: &[PathBuf]) -> Result<()> {
 
         debug!("Processing js file {}", js_path.display());
 
-        let f = File::open(js_path).context(format!("Failed to open {}", js_path.display()))?;
-        let f = BufReader::new(f);
+        let file =
+            fs::read_to_string(js_path).context(format!("Failed to open {}", js_path.display()))?;
 
-        let mut sourcemap_url = None;
-        for line in f.lines() {
-            let line = line.context(format!("Failed to process {}", js_path.display()))?;
-
-            if line.starts_with(DEBUGID_COMMENT_PREFIX) {
-                debug!("File {} was previously processed", js_path.display());
-                continue 'paths;
-            }
-
-            if let Some(url) = line.strip_prefix(SOURCEMAP_URL_COMMENT_PREFIX) {
-                sourcemap_url = Some(PathBuf::from(url));
-            }
+        if js::discover_debug_id(&file).is_some() {
+            debug!("File {} was previously processed", js_path.display());
+            continue 'paths;
```

Comment:
Do we need a lable here? Or is it just styling thing for you?

Reply:
That label is a vestige of a previously nested loop that I forgot to remove.